validator/client/validator_attest.go

```diff
@@ -99,18 +96,12 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		}
 		return
 	}
-
-	if featureconfig.Get().ProtectAttester {
-		if isNewAttSlashable(attesterHistory, data.Source.Epoch, data.Target.Epoch) {
-			log.WithFields(logrus.Fields{
-				"sourceEpoch": data.Source.Epoch,
-				"targetEpoch": data.Target.Epoch,
-			}).Error("Attempted to make a slashable attestation, rejected")
-			if v.emitAccountMetrics {
-				validatorAttestFailVec.WithLabelValues(fmtKey).Inc()
-			}
-			return
-		}
+	indexedAtt := &ethpb.IndexedAttestation{
+		AttestingIndices: []uint64{duty.ValidatorIndex},
+		Data:             data,
+	}
+	if err := v.preSigningValidations(ctx, indexedAtt, pubKey); err != nil {
+		return
```

Comment:
Same here for error handling

Reply:
i would like to leave it more granular (which service found it) WDYT?